doc/source/conf.py

```diff
@@ -217,9 +234,35 @@ def reset_servers(gallery_conf, fname, when):
         "min_chars_for_search": 2,
         "ignoreLocation": True,
     },
+    "ansys_sphinx_theme_autoapi": {
+        "project": project,
+        "output": "api",
+        "directory": "src/ansys",
+        "use_implicit_namespaces": True,
+        "keep_files": True,
+        "own_page_level": "class",
+        "type": "python",
+        "options": [
+            "members",
+            "undoc-members",
+            "show-inheritance",
+            "show-module-summary",
+            "special-members",
+        ],
+        "class_content": "class",
+        "ignore": autoapi_ignore_list,
+        "add_toctree_entry": True,
+        "member_order": "bysource",
+    }
 }
 
-
+# Configuration for Sphinx autoapi
+suppress_warnings = [
+    "autoapi.python_import_resolution",
```

Comment:
Why do we have to suppress this warning?

Reply:
Oh! I see... It is because we may be hitting cyclic imports. Could you add a TODO in here @moe-ad and open an issue so we do not forget about removing this in the future once the code is healthier?